flybywiresim/a32nx
# fix: aircraft States - fixed AUTOBRAKE setter LVAR name

Fixes #7272 

## Summary of Changes

- Fixes changed LVAR for setting AUROBRAKE
- Moves flypad-backend WASM into its own VCockpit instrument
- Small improvement to pushback counter rotation logic

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions

- Check that aircraft states correctly setup the aircraft - especially turning AUTOBRAKE to MAX in "Ready to Taxi" state or higher. 
- Make sure in general all wasm impacted functions still work (lighting presets, aircrafts states, pushback)
- Pushback should work as before - improvement for some rare situations were the front wheel was lifted while pulling the aircraft forward.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
